main.go

```diff
@@ -400,9 +430,39 @@ func (bot *bot) getRepositories(ctx context.Context, page, perPage int, affiliat
 	return bot.getRepositories(ctx, page, perPage, affiliation)
 }
 
-func (bot *bot) getIssues(ctx context.Context, page, perPage int, owner, repo string) error {
+func (bot *bot) getWatchedRepositories(ctx context.Context, page, perPage int, since time.Time) error {
+	opt := &github.ListOptions{
+		Page:    page,
+		PerPage: perPage,
+	}
+
+	repos, resp, err := bot.ghClient.Activity.ListWatched(ctx, "", opt)
+	if err != nil {
+		return err
+	}
+
+	for _, repo := range repos {
+		// logrus.Debugf("checking if %s is in (%s)", repo.GetOwner().GetLogin(), strings.Join(orgs, " | "))
+		// logrus.Debugf("getting issues for repo %s...", repo.GetFullName())
```

Comment:
you can uncomment these if it helps :)

Reply:
I updated the code to use the new cli format. After creating the PR, I didn't have the chance to use this app much. I think the rest is ok.